README.rst

```diff
@@ -95,7 +95,7 @@ corresponding image pull tags and their release notes where applicable:
 +----------------------------------------------------------------------------+------------+-----------------------------------------+------------------------------------------------------------------------------------------------+
 | `main <https://github.com/cilium/cilium/commits/main>`__                   | daily      | ``quay.io/cilium/cilium-ci:latest``     | N/A                                                                                            |
 +----------------------------------------------------------------------------+------------+-----------------------------------------+------------------------------------------------------------------------------------------------+
-| `v1.17.0-rc.2 <https://github.com/cilium/cilium/commits/v1.17.0-rc.2>`__   | 2025-01-24 | ``quay.io/cilium/cilium:v1.17.0-rc.2``  | `Pre Release Candidate Notes <https://github.com/cilium/cilium/releases/tag/v1.17.0-rc.2>`__   |
+| `v1.17.1-4-g05cb7b36e7 <https://github.com/cilium/cilium/commits/v1.17.1-4-g05cb7b36e7>`__   | 2025-02-17 | ``quay.io/cilium/cilium:v1.17.1-4-g05cb7b36e7``  | `Pre Release Candidate Notes <https://github.com/cilium/cilium/releases/tag/v1.17.1-4-g05cb7b36e7>`__   |
```

Comment:
I'd probably just skip adding this change to the commit.

I imagine the script just doesn't account for times like now where there has been the v1.17.0 and v1.17.1 releases but no v1.18.0-pre.0 yet. Rather than keeping the most recent prerelease (older than the latest patch releases) it seems to be trying to put together a dev version. This probably comes up twice a year immediately following a minor release.